app/com/linkedin/drelephant/tuning/IPSOConstants.java

```diff
@@ -0,0 +1,34 @@
+package com.linkedin.drelephant.tuning;
+
+public class IPSOConstants {
+
+  enum PARAMTER_CONSTRAINT {
```

Comment:
Can we rename to something like PARAMETER_NAME? This is independent of any constraints.